man/systemd.netdev.xml

```diff
@@ -315,6 +315,26 @@
           </listitem>
         </varlistentry>
         <varlistentry>
+          <term><varname>AgeingTimeSec=</varname></term>
+          <listitem>
+            <para>AgeingTimeSec specifies the number of seconds a MAC Address will be kept in
+            the forwaring database after having a packet received from this MAC Address.</para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term><varname>Priority=</varname></term>
+          <listitem>
+            <para>The priority of the bridge. An unsigned between 0 and 65535. A lower value
```

Comment:
The information that this value is unsigned is redundant, given you specify the range. Maybe say "An integer value between 0 and 65535".